pyanaconda/anaconda.py

```diff
@@ -229,6 +229,15 @@ def noninteractive_tui_mode(self):
         return (self._display_mode == DisplayModes.TUI
                 and not self._interactive_mode)
 
+    def report_display_mode(self):
```

Comment:
I would rather name this `log_display_mode()`. Report can be anything, log is unambiguous.

Reply:
Fixed.